app/javascript/components/public_health/PatientsTable.js

```diff
@@ -91,55 +91,76 @@ class PatientsTable extends React.Component {
   }
 
   componentDidMount() {
-    // load saved tab from local storage if present
+    // load local storage variables when present
+    const query = {};
+
+    // Set tab from local storage if it exists and is a valid tab
     let tab = localStorage.getItem(`${this.props.workflow}Tab`);
-    if (tab === null || !(tab in this.props.tabs)) {
-      tab = this.state.query.tab;
-      localStorage.setItem(`${this.props.workflow}Tab`, tab);
+    if (tab === null || !Object.keys(this.props.tabs).includes(tab)) {
+      query.tab = this.state.query.tab;
+      localStorage.setItem(`${this.props.workflow}Tab`, query.tab);
+    } else {
+      query.tab = tab;
     }
 
-    // select tab and fetch patients
-    this.handleTabSelect(tab);
-
     // Set jurisdiction if it exists in local storage
     let jurisdiction = localStorage.getItem('SaraJurisdiction');
     if (jurisdiction) {
-      this.handleJurisdictionChange(parseInt(jurisdiction));
+      query.jurisdiction = parseInt(jurisdiction);
+    }
+
+    // Set scope if it exists in local storage
+    let scope = localStorage.getItem('SaraScope');
+    if (scope) {
+      query.scope = scope;
     }
 
     // Set assigned user if it exists in local storage
     let assigned_user = localStorage.getItem('SaraAssignedUser');
     if (assigned_user) {
-      this.handleAssignedUserChange(assigned_user);
+      query.user = assigned_user;
     }
 
     // Set search if it exists in local storage
     let search = localStorage.getItem(`SaraSearch`);
     if (search) {
-      this.setState(
-        state => {
-          return {
-            query: { ...state.query, search: search },
-          };
-        },
-        () => {
-          this.updateTable(this.state.query);
-        }
-      );
+      query.search = search;
     }
 
-    // Select page if it exists in local storage
+    // Set page & sort settings if they exist in local storage & user is in the same workflow as before
+    let priorWorkflow = localStorage.getItem(`Workflow`);
     let page = localStorage.getItem(`SaraPage`);
-    if (page) {
-      this.handlePageUpdate(JSON.parse(page));
+    let sortField = localStorage.getItem(`SaraSortField`);
+    let sortDirection = localStorage.getItem(`SaraSortDirection`);
+    if (priorWorkflow && this.props.workflow === priorWorkflow) {
```

Comment:
I think you may need to also check if the tab has changed here, because if it does we want to remove the page, sort, and direction items as well (as is done in `handleTabUpdate()` which was previously called in this method instead).

Reply:
do we also want to check for jurisdiction/scope/assigned_user/search/advanced filter change here as well? I think if any of these have changed, it might make sense to just revert to the first page

Reply:
@kylietmo That makes sense to me, I will look to include that additional check. 

@timwongj The clearing out of the page sticky setting for a change in  jurisdiction/scope/assigned_user/search/advanced filter is already done in each of their respective handleChange methods. So I don't think checks for those are needed here. If I misunderstood your point though, feel free to clarify.

Reply:
After discussing this with Becky, we decided that the use case I mentioned above is handled by `handleTabUpdate` and does not need to be handled in `componentDidMount`. We do still think sorting and sorting direction should also be reset in in `handleTabUpdate`. 

@timwongj Given that this is on component mount, the only previous values we can grab are from localStorage, so if they had changed, those should have been handled in their specific change handlers.

Reply:
@kylietmo that makes sense, thanks!